NETWORK/NetworkFadeInEntity.md

```diff
@@ -5,20 +5,31 @@ ns: NETWORK
 
 ```c
 // 0x1F4ED342ACEFE62D 0x9B9FCD02
-void NETWORK_FADE_IN_ENTITY(Entity entity, BOOL state);
+void NETWORK_FADE_IN_ENTITY(Entity entity, BOOL flash, BOOL slow);
```

Comment:
At this time adding new arguments in this fashion in not supported. Maybe make a note of it in the native description? Thanks!